test/flink_test.go

```diff
@@ -36,6 +36,7 @@ func (s *CLITestSuite) TestFlinkArtifact() {
 	tests := []CLITest{
 		{args: "flink artifact create my-flink-artifact --artifact-file test/fixtures/input/flink/java-udf-examples-3.0.jar", fixture: "flink/artifact/create.golden"},
 		{args: "flink artifact create my-flink-artifact --artifact-file test/fixtures/input/flink/java-udf-examples-3.0.jar --description cliPluginTest", fixture: "flink/artifact/create.golden"},
+		{args: "flink artifact create my-flink-artifact --artifact-file test/fixtures/input/flink/python-udf-examples.zip --description cliPluginTest --runtime-language python", fixture: "flink/artifact/create-python.golden"},
```

Comment:
@byang-confluent  let me know if you want me to remove this one as well.

Reply:
this is a test case, correct? i think it's ok to create a test case for python